File: src/main/java/net/lz1998/pbbot/SpringMiraiServerApplication.java
package net.lz1998.pbbot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.EnableAspectJAutoProxy;

@EnableAspectJAutoProxy
@SpringBootApplication
public class SpringMiraiServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(SpringMiraiServerApplication.class, args);
    }

}

File: src/main/kotlin/net/lz1998/mirai/SpringMiraiServerApplication.kt
package net.lz1998.mirai

import net.lz1998.mirai.bot.EventProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties(EventProperties::class)
class SpringMiraiServerApplication

fun main(args: Array<String>) {
    runApplication<SpringMiraiServerApplication>(*args)
}
